code/modules/admin/admin_verbs.dm

```diff
@@ -60,6 +60,7 @@ var/list/admin_verbs_admin = list(
 	/client/proc/toggledrones,
 	/client/proc/man_up,
 	/client/proc/global_man_up,
+	/client/proc/write_RD_json,

```

Comment:
Argh, I actually forgot that this was in the pull request. I'll add it to the main description.